index.mako

```diff
@@ -1103,6 +1103,8 @@ pre-commit installed at .git/hooks/commit-msg
 current contents of the commit message which can be validated.  If a hook
 exits nonzero, the commit will be aborted.
 
+_note_: you may need to provide `--commit-msg-filename` when testing this hook type.
```

Comment:
I think this probably makes more sense in the `pre-commit try-repo` / `pre-commit run` sections above

Reply:
Good point. I made this change.